### ISSUE TITLE
Unsupervised

### ISSUE THREAD

**Issue by joyjoy2026** [JiangtaoNie/UAL#3]
Hi, My name is ZhuangChen from HUST, I am reading your paper named "Unsupervised Adaptation Learning for Hyperspectral Imagery Super-resolution" and getting a lots of knowledge. Really thk U!  While I have a question about your paper   . 

U say "a pre-trained fusion module whose weights are fixed in unsupervised learning" in section 4.2. But when I am reading your code in "Train_FusionModel.py"  I find that U still use loss function to compute loss and use optimizer to update parameters as other supervised algorithm. 

 I feel so confused about that. Can u help me work this question out ? Thks!

**Comment by JiangtaoNie** [comment]
The "Train_FusionModel.py" is the code to pre-train the fusion model, we train it to exploit the general priors that contain in external datasets in a supervised manner. The training of adapter network is totally unsupervised.

**Comment by Dou0405** [comment]
您好，我很困惑在adapter network的训练中，这里优化目标Z_i具体是什么，M数据集是怎样生成的？

![Image](https://github.com/user-attachments/assets/e7aaeb8c-5c29-48dd-b505-fb6526c30f82)

**Comment by JiangtaoNie** [comment]
这里是想通过元学习来预训练我们的Adaptor，所以这里面用的还是训练集里面的数据，结合不同的降质来使得模型可以学习不同降质之间的差异。Z_i就是GT，数据集中的第i个数据，j表示不同的降质。

**Comment by Dou0405** [comment]
这个信息太重要了，非常感谢您的解答，祝多发TPAMI
